src/OrbitGl/App.h

```diff
@@ -585,6 +585,9 @@ class OrbitApp final : public DataViewFactory,
   void ShowHistogram(const std::vector<uint64_t>* data, const std::string& scope_name,
                      uint64_t scope_id) override;
 
+  void RequestSymbolDownloadStop(absl::Span<const orbit_client_data::ModuleData* const> modules,
```

Comment:
Nit: I would prefer to have show_dialog defaulted to true here instead of the overload (which I found more difficult in term of understanding the code), but I suppose you considered this, so take it as a nit.

Reply:
Yeah, no default parameters with virtual functions :/